guessingGame/GuessingGame.cs

```diff
@@ -0,0 +1,155 @@
+using System;
```

Comment:
I cannot build this project. I get the following errors when I attempt to `dotnet run`.

```
GuessingGame.cs(11,29): error CS0246: The type or namespace name 'CheatController' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\daniel.ferguson\Downloads\ACA\Brett\guessingGame\guessingGame.csproj]
GuessingGame.cs(41,61): error CS0246: The type or namespace name 'CheatController' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\daniel.ferguson\Downloads\ACA\Brett\guessingGame\guessingGame.csproj]
PlayController.cs(11,9): error CS0246: The type or namespace name 'CheatController' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\daniel.ferguson\Downloads\ACA\Brett\guessingGame\guessingGame.csproj]
GuessingGame.cs(39,17): error CS0246: The type or namespace name 'CheatController' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\daniel.ferguson\Downloads\ACA\Brett\guessingGame\guessingGame.csproj]

The build failed. Please fix the build errors and run again.
```